### PR TITLE
bt-582: [BUG] Validation schema differs from frontend to backend

### LINKED ISSUE #582: [BUG] Validation on BE doesn't match the frontend
**Description:**
Validation on BE doesn't match the frontend and validation doc

**Environment:**
Win 11, Chrome 116

**Steps to reproduce:**
1. Fill in all fields with valid data
2. Click Submit

**Actual Result:**
1. Error on company website and description

**Expected Result:**
1. Company website should have validation as in the Validation doc. www.ex.ua should be considered as a valid url.
2. Description field can be empty

**Visual Proof (screenshots, videos, text):**
Image

**Severity:**
Major

### PR DESCRIPTION
**Pull request by lavandosya**

### CODE REVIEW

File: shared/src/bundles/user-details/validation-schemas/user-details-create.validation-schema.ts [hunk 1]
```diff
@@ -61,7 +61,9 @@ const userDetailsCreate = joi.object<UserDetailsCreateRequestDto, true>({
     linkedinLink: joi.string().trim().uri(),
     companyName: joi.string().trim(),
     companyLogoId: joi.string().trim(),
-    companyWebsite: joi.string().trim().uri(),
+    companyWebsite: joi
+        .string()
+        .regex(/^(www\.|http:\/\/|https:\/\/)[^.]+(\..+)+$/),
```
Comment (anton-otroshchenko) [inline shared/src/bundles/user-details/validation-schemas/user-details-create.validation-schema.ts]: Create a variable for it

**Review by anton-otroshchenko — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/employer-onboarding/components/onboarding-form/onboarding-form.tsx [hunk 2]
```diff
@@ -354,6 +354,7 @@ const OnboardingForm: React.FC = () => {
                             <Typography variant="label">
                                 Briefly tell about your company and its values
                             </Typography>
+                            <span className={styles.requiredField}>*</span>
```
Comment (Monyster) [inline frontend/src/bundles/employer-onboarding/components/onboarding-form/onboarding-form.tsx]:
```suggestion
```

Comment (Monyster) [reply frontend/src/bundles/employer-onboarding/components/onboarding-form/onboarding-form.tsx]: add require to FormLabel

File: frontend/src/bundles/employer-onboarding/enums/onboarding-form/onboarding-form.validation-rule.ts [hunk 3]
```diff
@@ -12,6 +12,7 @@ const EmployerOnboardingValidationRule = {
     MIN_COMPANY_NAME_LENGTH: 2,
     MAX_COMPANY_NAME_LENGTH: 50,
     MAX_FILE_SIZE: 5_242_880,
+    URL_REGEX_CONSTANT: /^(www\.|http:\/\/|https:\/\/)[^.]+(\..+)+$/,
 };
```
Comment (Monyster) [inline frontend/src/bundles/employer-onboarding/enums/onboarding-form/onboarding-form.validation-rule.ts]:
```suggestion
} as const;
```

File: frontend/src/bundles/employer-onboarding/validation-schemas/onboarding-form/onboarding-form.validation-schema.ts [hunk 4]
```diff
@@ -131,13 +131,19 @@ const EmployerOnboardingValidationSchema = joi.object<
 
     description: joi
         .string()
-        .trim()
+        .required()
```
Comment (Monyster) [inline frontend/src/bundles/employer-onboarding/validation-schemas/onboarding-form/onboarding-form.validation-schema.ts:135]:
```suggestion
        .trim().required()
```

**Review by Monyster — CHANGES_REQUESTED** [review]

**Review by Monyster — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**